template.yaml

```diff
@@ -629,3 +648,14 @@ Resources:
       Runtime: java17
       MemorySize: 1024
       Role: !GetAtt SearchHandlerRole.Arn
+
+  KeyBatchesBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      AccessControl: Private
+      BucketName: !Sub "${KeyBatchesBucketName}-${AWS::AccountId}"
+      LifecycleConfiguration:
+        Rules:
+          - Id: DeleteContentAfter3Days
+            Status: Enabled
+            ExpirationInDays: 3
```

Comment:
NB! make sure this is not too short span. We have had issues with this in cristin-import.